websites/F/Flat/presence.ts

```diff
@@ -0,0 +1,84 @@
+const presence = new Presence({
+	clientId: "1021848171946704906",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "flat",
```

Comment:
Use an Imgur link for the `largeImageKey` rather than Discord asset names.

Reply:
will do tomorrow

Reply:
```suggestion
			largeImageKey: "https://i.imgur.com/pFfJvOZ.png",
```

Reply:
will export as enum instead